packages/core-filters/director.js

```diff
@@ -65,6 +80,16 @@ class FilterDirector {
 
   async search(productIdResolver, options = {}) {
     return this.reduceAdapters(async (lastSearchPromise, concreteAdapter) => {
+      // concreteAdapter.context.query.assortmentIds
+      if (
+        concreteAdapter.__proto__.constructor.name === 'LocalSearch' &&
```

Comment:
It's not guaranteed that LocalSearch plugin is there and it's also not guaranteed that the user wants to use the LocalSearch plugin to search assortments

Reply:
I agree with your concerns but the alternative would be to pull out `searchAssortments` code into some other place and I thought it'd be nice to keep the search functionality in one place.